voxxed-microservices-2018/module-01.adoc

```diff
@@ -238,6 +238,16 @@ In addition to the `cluster operator` created previously, notice a few more depl
 
 Also, notice that the Zookeeper ensemble and the Kafka cluster are deployed as stateful sets.
 
+=== Resource limits
+The deployed Kafka broker instances and the ZooKeeper are by defualt configured to use all available memory provided by the environment.
+This is usually undesirable so we will set OpenShift limits to them so each of the pods will receive only allocated portion of memory which is 1 GB in case of each broker instance and 512 MB in case of ZooKeeper.
```

Comment:
Could we limit it to 512 MB for Kafka, too? It'd be nice if we could keep everything together below 4 GB.

Reply:
Ok, makes sense. Thx!